gulp-ng-config.js

```diff
@@ -104,7 +104,7 @@ function gulpNgConfig (moduleName, configuration) {
     _.each(jsonObj, function (value, key) {
       constants.push({
         name: key,
-        value: JSON.stringify(value, null, spaces)
+        value: JSON.stringify(value, null, spaces).replace(/"/g, '\'')
```

Comment:
What if some of the data contains quotes? 

For example:

```js
var input = {
  text: 'The person said "Hello"'
}

JSON.stringify(input)
// '{"text":"The person said \"Hello\" loudly"}'

JSON.stringify(input).replace(/"/g, '\'')
// '{\'text\':\'The person said \'Hello\' loudly\'}'
```

That would feel like unexpected behavior to modify the contents